test/test_logging_bad_env.cpp

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+
+#include "rcutils/logging.h"
+
+TEST(TestBadCustomEnv, test_initialize) {
+  EXPECT_FALSE(g_rcutils_logging_initialized);
+  ASSERT_EQ(RCUTILS_RET_INVALID_ARGUMENT, rcutils_logging_initialize());
```

Comment:
@brawner uh, `RCUTILS_RET_INVALID_ARGUMENT` for a function that takes no arguments?

Reply:
They are set in the CMakeLists.txt file. But `rcutils_logging_initialize()` checks environment variables and will return an invalid argument in that case.

Reply:
I see. It's out of scope for this PR, but I find the error code selection weird. Environment variables are not function arguments.